munimap/static/js/geoeditor/geoeditor-draw-controller.js

```diff
@@ -169,10 +169,21 @@ angular.module('munimapGeoeditor')
 
                     feature.set('style', style);
                 }
-
+                
                 $rootScope.$broadcast('geoeditor:openPopupFor', layer, feature);
             };
 
+            // This following event is used to trigger an update of the popup configuration
+            // when clicking on an existing geographic element on the map.
+            // Without this event the popup element will show the configuration of the last
+            // created element, i.e, it can show tabs for content that should be shown, like the attribute form, 
+            // even when there are no attributes present.
+            $olOn(MapService.getMap(), 'singleclick', (evt) => {
```

Comment:
Does this need to be unregistered at some point?

Reply:
I don't think it needs to be unregistered as it is not possible to deactivate the geoeditor at any point.

Reply:
If the geoeditor is not configured, the listener won't be registered.